<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manual</class>
 <widget class="QDialog" name="Manual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manuelle Bedienung</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="Btn_down">
     <property name="text">
      <string>down</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="Btn_up">
     <property name="text">
      <string>up</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Btn_right">
     <property name="text">
      <string>right</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="Btn_left">
     <property name="text">
      <string>left</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="Btn_close">
     <property name="text">
      <string>Verstecken</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Btn_up</tabstop>
  <tabstop>Btn_down</tabstop>
  <tabstop>Btn_left</tabstop>
  <tabstop>Btn_right</tabstop>
  <tabstop>Btn_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Manual</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
